phoenix-core/src/main/java/org/apache/phoenix/jdbc/LoggingPhoenixPreparedStatement.java

```diff
@@ -17,20 +17,20 @@
  */
 package org.apache.phoenix.jdbc;
 
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.SQLFeatureNotSupportedException;
+import java.sql.*;
 
 public class LoggingPhoenixPreparedStatement extends DelegatePreparedStatement {
     
     private PhoenixMetricsLog phoenixMetricsLog;
     private String sql;
+    private Connection conn;
     
-    public LoggingPhoenixPreparedStatement(PreparedStatement stmt, PhoenixMetricsLog phoenixMetricsLog, String sql) {
+    public LoggingPhoenixPreparedStatement(Connection conn, PreparedStatement stmt,
+                                           PhoenixMetricsLog phoenixMetricsLog, String sql) {
         super(stmt);
         this.phoenixMetricsLog = phoenixMetricsLog;
         this.sql = sql;
+        this.conn = conn;
```

Comment:
nit: Can you move the init of variables in the same order as function arguments?